CoffeeScript.tmLanguage

```diff
@@ -222,12 +222,32 @@
 			<key>name</key>
 			<string>string.regexp.coffee</string>
 		</dict>
+		<dict>
+            <key>comment</key>
+            <string>Show for-from as keyword.control</string>
+			<key>captures</key>
+			<dict>
+				<key>1</key>
+				<dict>
+					<key>name</key>
+					<string>keyword.control.coffee</string>
+				</dict>
+				<key>2</key>
+				<dict>
+					<key>name</key>
+					<string>keyword.control.coffee</string>
+				</dict>
+			</dict>
+			<key>match</key>
+			<string>\b(?:(for)(?:\b.+\b)(from))\b</string>
```

Comment:
This doesn’t appear to catch all uses. In particular I’m looking in the CoffeeScript repo:

https://github.com/jashkenas/coffeescript/blob/c4245e50c2fda100c258110727ac8e76d2ac1a92/test/arrays.coffee#L252-L289

I don’t see any of these `from` keywords highlighted (though I do see `await` highlighted elsewhere, so I should have your fork loaded).